### PR TITLE
Incorporate relevant 2.x changes into 3.x

### LINKED ISSUE #230: [3.0.0] Tooltip content is not wormholed on ember-wormhole 0.5.4
Hi,
After updating to ember-wormhole 0.5.4 tooltip content is not properly wormholed - it shows just under the trigger element, not in the wormhole container. It seems that this is related PR:
https://github.com/yapplabs/ember-wormhole/pull/101

Regards!

### LINKED ISSUE #259: Bug: Ember Wormhole Is A Dev Dependency (3.0.0-beta.2)
Ember wormhole is a development dependency and is not installed in host application

### PR DESCRIPTION
**Pull request by maxfierke**
Updates the 3.x branch with the ember-cli updates from the master, as well as incorporating the changes made to 2.x since 3.x was branched off.

Notably:

- [x] Ember 3.x support
- [x] Replaces SASS/SCSS with plain CSS
- [x] Removes deprecated use of `sendAction`
- [x] Fixes tooltip rendering under `ember-wormhole` 0.5.4 (Fixes #230, side-effect fixes #259)
- [x] Fixes CI build for 3.x

### CODE REVIEW

File: tests/dummy/app/controllers/index.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
-import Ember from 'ember';
-
-const {
-  Controller,
-  RSVP,
-  run,
-} = Ember;
+import Controller from '@ember/controller';
+import RSVP from 'rsvp';
+import { run } from '@ember/runloop';
```
Comment (sir-dunxalot) [inline tests/dummy/app/controllers/index.js]: You can do `import { later, scheduleOnce } from '@ember/runloop';`

File: tests/helpers/destroy-app.js [hunk 2]
```diff
@@ -1,6 +1,4 @@
-import Ember from 'ember';
```
Comment (sir-dunxalot) [inline tests/helpers/destroy-app.js]: If you decide we should update the test suite as a part of this PR, no Ember blueprint helpers need to be left in the `tests` dir.

File: tests/acceptance/acceptance-test.js [hunk 3]
```diff
@@ -10,8 +10,7 @@ import {
   assertTooltipVisible,
 } from '../../tests/helpers/ember-tooltips';
 
-const { $ } = Ember;
-
+/* globals andThen, visit */
```
[the remark this answers is not in the document]
Comment (maxfierke) [reply tests/acceptance/acceptance-test.js]: Yeah, I think that's probably sizable enough for a separate PR (should be mostly automated too, with the codemods)

File: index.js [hunk 4]
```diff
@@ -26,7 +25,7 @@ module.exports = {
   },
 
   included: function(app) {
-    this._super.included(app); // For ember-cli-sass
+    this._super.included(app);
```
[the remark this answers is not in the document]
Comment (maxfierke) [reply index.js]: Might not strictly be, but it's documented in a basic hook implementation in the [ember-cli docs](https://ember-cli.com/api/classes/Addon.html#method_included) and seems like it's probably good form when overriding hooks.

[the remark this answers is not in the document]
Comment (sir-dunxalot) [reply index.js]: Sounds good.

**Pull request merged**